sdk/identity/azure-identity/azure/identity/_credentials/user.py

```diff
@@ -193,12 +201,11 @@ class UsernamePasswordCredential(PublicClientCredential):
     :param str password: the user's password
 
     Keyword arguments
-        - **authority**: Authority of an Azure Active Directory endpoint, for example 'login.microsoftonline.com', the
-          authority for Azure Public Cloud (which is the default). :class:`~azure.identity.KnownAuthorities` defines
-          authorities for other clouds.
-        - **tenant (str)** - tenant ID or a domain associated with a tenant. If not provided, defaults to the
+        - **authority** (str): Authority of an Azure Active Directory endpoint, for example 'login.microsoftonline.com',
+          the authority for Azure Public Cloud (which is the default). :class:`~azure.identity.KnownAuthorities`
+          defines authorities for other clouds.
+        - **tenant_id** (str) - tenant ID or a domain associated with a tenant. If not provided, defaults to the
```

Comment:
I think tenant_id should still be required here. No?

Reply:
I tested a public application granted user_impersonation to Key Vault. The credential can authenticate a user against the 'organizations' tenant and get valid access tokens.